process/factory/shard/vmContainerFactory.go

```diff
@@ -79,9 +79,9 @@ func (vmf *vmContainerFactory) Create() (process.VirtualMachinesContainer, error
 }
 
 func (vmf *vmContainerFactory) createArwenVM() (vmcommon.VMExecutionHandler, error) {
-	if vmf.config.OutOfProcessEnabled {
-		return vmf.createOutOfProcessArwenVM()
-	}
+	// if vmf.config.OutOfProcessEnabled {
```

Comment:
commented code - without TODO

Reply:
Fixed (was for debugging the MacOS build).